general/Jupyterhub/jupyter/sbatch_jupyter.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash  --login
 
 ## Modify walltime and account at minimum
-#SBATCH --time=01:00:00
-#SBATCH --account=<project handle>
+#SBATCH --time=<time_estimate>
+#SBATCH --account=<project_handle>
 
 #SBATCH --nodes=1
 #SBATCH --tasks-per-node=1
 #SBATCH --cpus-per-task=36
```

Comment:
Core number should be updated for Kestrel nodes. We could just make this a script to run on shared nodes, and add on the documentation page which values need to be updated for a job, and how to optionally make it run on a full node instead

Reply:
Okay, working on writing separate scripts right now. When they are ready for review, I'll change it from draft status to review status.